runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1271,6 +1300,50 @@ gcParseReconfigurableArguments(J9JavaVM* vm, J9VMInitArgs* args)
 
 	return true;
 
+_error:
+	return false;
+}
+/**
+ * Parse command line arguments used ONLY for CRIU restore. Note the arguments
+ * are now passed in through an option file.
+ *
+ * @return true if parsing was successful, false otherwise.
+ */
+bool
+gcParseReconfigurableCommandLine(J9JavaVM* vm, J9VMInitArgs* args)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(vm);
+	IDATA index = -1;
+	IDATA result = 0;
+	PORT_ACCESS_FROM_JAVAVM(vm);
+	if (-1 != FIND_ARG_IN_ARGS(args, EXACT_MEMORY_MATCH, VMOPT_XSOFTMX, NULL)) {
+		result = option_set_to_opt_args(vm, VMOPT_XSOFTMX, &index, EXACT_MEMORY_MATCH, &extensions->softMx, args);
```

Comment:
let's be consistent throughout the file and use OPT_SOFTMX at these 2 spots too

Reply:
fixed, thanks.